packages/fx-core/tests/plugins/resource/sql/unit/generateArmTemplates.test.ts

```diff
@@ -104,7 +104,7 @@ describe("generateArmTemplates", () => {
       version: "1.0.0",
       activeResourcePlugins: activeResourcePlugins,
     } as AzureSolutionSettings;
-    const result = await sqlPlugin.generateArmTemplates(pluginContext);
+    const result = await sqlPlugin.updateArmtemplates(pluginContext);
 
     // Assert
     const testModuleFileName = "sqlProvision.result.bicep";
```

Comment:
you may also need to remove L125 when updating the test case

Reply:
sure, thanks